admin/class-acc_user_importer-admin.php

```diff
@@ -356,94 +301,58 @@ private function proccess_user_data ( $users ) {
 			$userCity = $user["City"] ?? '';
 			
 			//Log the info we received for this user
-			$user_info = $userContactId;
-			$user_info = $user_info . " " . $userEmail;
-			$user_info = $user_info . " home:" . $userHomePhone;
-			$user_info = $user_info . " cell:" . $userCellPhone;
-			$user_info = $user_info . " member#:" . $userMembership;
-			$user_info = $user_info . " expiry:" . $userExpiry;
-			$this->log_dual("Received " . $user_info);
+			$userInfoString = $userContactId;
+			$userInfoString .= " " . $userEmail;
+			$userInfoString .= " home:" . $userHomePhone;
+			$userInfoString .= " cell:" . $userCellPhone;
+			$userInfoString .= " member#:" . $userMembership;
+			$userInfoString .= " expiry:" . $userExpiry;
+			$this->log_dual("Received " . $userInfoString);
+
+
+			// Skip users without membership number
+			if (!is_numeric($userMembership) || !is_numeric($userContactId)) {
+				$this->log_dual("User is missing membership number or contact ID, skipping.");
+				continue;
+			}
```

Comment:
This seems like the right thing to do because I have seen over the last months a few cases where a user would have duplicated entries in the National website, one with a userMembership, and one without, and in such case we should ignore the one without userMembership.  When I noticed this, I talked to Jeff who is the tech guy for the ACC database and he fixed things (removed the entries without userMembership).